HelpSource/Classes/File.schelp

```diff
@@ -40,7 +40,22 @@ File.getcwd;
 ::
 
 method::use
-open the file, evaluate the function with the file and close it.
+open the file, evaluate the function with the file as argument, and close it again. If the process fails, close the file and throw an error.
+
+method::read
+given a path, make a new temporary file, and read the contents, close the file and return them. By default, it calls link::#-readAllString::. If the process fails, close the file and throw an error.
+
+argument::pathName
+the path (link::Classes/String::) that points to the file
+
+argument::mode
+the file mode to use.
+
+argument::selector
+the method to be called on the file once it is open.
+
+
```

Comment:
What are the valid methods that could be supplied here ?  \readAllString ,  ... ?